test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
@ryanbrandenburg - how the MVC are being picked up during build? At the moment build.cmd does not seem to recognize `netcoreapp1.0` test projects: https://github.com/aspnet/KoreBuild/blob/dev/build/shade/_k-standard-goals.shade#L227-L230 (not a big deal to change it and I did this locally when I tried running netcoreapp1.0 tests but hit a different issue) https://github.com/aspnet/KoreBuild/blob/dev/build/shade/_k-standard-goals.shade#L227-L230